pom.xml

```diff
@@ -177,6 +177,9 @@
         <!-- Reactive Streams version -->
         <reactive-streams.version>1.0.4</reactive-streams.version>
 
+        <!-- S3 Access Grants plugin version -->
+        <s3accessgrants.version>2.0.0</s3accessgrants.version>
```

Comment:
Will wait until the patch release to remove the test dependency before merging.

Reply:
Thanks @millems, bumped up the version.